sentinel-core/src/test/java/com/alibaba/csp/sentinel/util/ConfigUtilTest.java

```diff
@@ -79,6 +81,45 @@ public void testLoadProperties() throws IOException {
 
     }
 
+    //add Jvm parameter
+    //-Dcsp.sentinel.charset="GBK"
+    //@Test
+    public void testLoadPropertiesWithCustomizedCharset() throws IOException {
 
+        String charset = "GBK";
+
+        File file = null;
+        String dir = "/data/日志/",
+                fileName = "propertiesTest.properties";
```

Comment:
I think the `dir` had better not included  Chinese character. Because some computer os may not support Chinese.

Reply:
👍